src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule3sourcefile/InputFormattedSourceFileStructure.java

```diff
@@ -0,0 +1,111 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code and other text files for adherence to a set of rules.
+//
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+///////////////////////////////////////////////////////////////////////////////////////////////
+package com.google.checkstyle.test.chapter3filestructure.rule3sourcefile;
+
+// violation 2 lines above ''package' should be separated from previous line.'
```

Comment:
Bug found in formatter, [3 Source file structure](https://google.github.io/styleguide/javaguide.html#s3-source-file-structure) says:

```
A source file consists of, in order:

License or copyright information, if present
Package statement
Import statements
Exactly one top-level class

Exactly one blank line separates each section that is present.
```

So there should be one blank line separating copyright/license information and package statement but formatter didn't included it and our config was complaining about it